editor/gui/editor_toaster.cpp

```diff
@@ -371,6 +371,20 @@ Control *EditorToaster::popup(Control *p_control, Severity p_severity, double p_
 	p_control->set_h_size_flags(SIZE_EXPAND_FILL);
 	hbox_container->add_child(p_control);
 
+	// Copy button.
+	if (p_time > 0.0) {
+		Button *copy_button = memnew(Button);
+		copy_button->set_flat(true);
+		copy_button->set_button_icon(get_editor_theme_icon(SNAME("ActionCopy")));
+		copy_button->set_focus_mode(FOCUS_NONE);
+		copy_button->set_disabled(true);
+		copy_button->connect(SceneStringName(mouse_entered), callable_mp(this, &EditorToaster::_toggle_copy_button).bind(copy_button, false));
+		copy_button->connect(SceneStringName(mouse_exited), callable_mp(this, &EditorToaster::_toggle_copy_button).bind(copy_button, true));
+		copy_button->connect(SceneStringName(pressed), callable_mp(this, &EditorToaster::copy).bind(panel));
+		copy_button->connect(SceneStringName(theme_changed), callable_mp(this, &EditorToaster::_copy_button_theme_changed).bind(copy_button));
```

Comment:
You can unify theme changeds in one method and bind both buttons.

Reply:
I opened #98794

Reply:
Awesome, thanks dude. I'll change stuff when that is merged.

Reply:
It was merged.

Reply:
Rebased and updated, thanks again.